pandas/tests/scalar/timedelta/test_construction.py

```diff
@@ -210,3 +206,28 @@ def test_td_constructor_on_nanoseconds(constructed_td, conversion):
 def test_td_constructor_value_error():
     with pytest.raises(TypeError):
         Timedelta(nanoseconds='abc')
+
+
+class not_raises(object):
+    def __enter__(self):
+        pass
```

Comment:
remove this its not idiomatic

Reply:
I kinda needed it for parametrizing on the expectation. Should I then just add a if/else in the test to use or not pytest.raises?